_rules/auto-play-audio-has-control-mechanism-95e043.md

```diff
@@ -0,0 +1,226 @@
+---
+id: 95e043
+name: auto-play audio has a control mechanism
+rule_type: atomic
+description: |
+  auto-play audio or video must have a control mechanism.
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>`, `<video>` or `<source>` elements, with a `src` attribute referencing content with a duration of more than 3 seconds, that has an `autoplay` attribute equal to `true`, and that has both `paused` and `muted` attributes equal to `false`.
+
+## Expectation 1
+
+For each test target a control mechanism is provided to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Expectation 2
+
+The control mechanism to pause or stop or turn the audio volume off is [visible](#visible), has an [accessible name](#accessible-name), and is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+The native `<video>` and `<audio>` controls in several browser and assistive technology combinations are not keyboard accessible and the `<video>` or `<audio>` element itself may not be anounced. Authors are recomended to use custom controls for keyboard navigation and cross browser accessibility support in general.
+
+## Background
+
+- [Understanding Success Criterion 1.4.2: Audio Control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html)
+- [Failure of Success Criterion 1.4.2 for absence of a way to pause or stop an HTML5 media element that autoplays](https://www.w3.org/WAI/WCAG21/Techniques/failures/F93)
+- [G170: Providing a control near the beginning of the Web page that turns off sounds that play automatically](https://www.w3.org/WAI/WCAG21/Techniques/general/G170)
+
+## Test Cases
+
+### Passed
+
+### Passed example 1
+
+The `<audio>` element has a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+#### Passed example 2
```

Comment:
correct number of hashes here, but other examples need to be updated to have 4 hashes.

Reply:
Nice catch. fixed this.